sys_path_hacks/lms/__init__.py

```diff
@@ -1,4 +1,5 @@
-import warnings
-warnings.warn("Importing lms.djangoapps instead of lms.djangoapps is deprecated", stacklevel=2)
+from sys_path_hacks.warn import warn_deprecated_import
+
+warn_deprecated_import('lms.djangoapps', 'lms.djangoapps')
```

Comment:
This is a bit odd. It was a bit odd before, too.

Reply:
Ha. That is funny. I'll update the script to remove that.